CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+# [1.29.0 (October 26th, 2023)](https://github.com/eclipse/eclipse.jdt.ls/milestone/120?closed=1)
+ * enhancement - Support named pipes for client/server communication. See [#2922](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2922).
+ * enhancement - Add new alias `public static void main(String[] args)` for public main method. See [#2919](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2919).
+ * enhancement - Add `print` alias for `sysout` snippet. See [#2918](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2918).
```

Comment:
This feature has been replaced by https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2927.  Maybe it's better to link to the new PR https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2927.